modules/addScore.js

```diff
@@ -6,7 +6,7 @@ export default function addScore(scores) {
 
```

Comment:
- [ ] In this file kindly change `addScore` to an arrow function as suggested by the reviewer. All functions should be arrow functions, and should not use the `function` keyword.

  ![image](https://github.com/Luffytaro22/Leaderboard/assets/54848363/39ec111f-e334-4861-a501-412bd97e15bb)